### ISSUE TITLE
 join() missing 1 required positional argument: 'a'

### ISSUE THREAD

**Issue by ThomasWaldmann** [borgbackup/borg#3388]
```
    xenial64:     source = os.path.join(*item.source.split(os.sep)[stripped_components:])
    xenial64: TypeError: join() missing 1 required positional argument: 'a'
    xenial64: 
    xenial64: Platform: Linux ubuntu-xenial 4.4.0-97-generic #120-Ubuntu SMP Tue Sep 19 17:28:18 UTC 2017 x86_64 x86_64
    xenial64: Linux: Ubuntu 16.04 xenial
    xenial64: Borg: 1.1.3  Python: CPython 3.5.2
```

Sometimes I've seen this on travis, for python 3.6-dev, but now also seen with non-dev python.

**Comment by ThomasWaldmann** [comment]
```
    jessie64: TypeError: join() missing 1 required positional argument: 'a'
    jessie64: 
    jessie64: Platform: Linux jessie 3.16.0-4-amd64 #1 SMP Debian 3.16.43-2+deb8u2 (2017-06-26) x86_64 
    jessie64: Linux: debian 8.9 
    jessie64: Borg: 1.1.3  Python: CPython 3.4.2
```

**Comment by ThomasWaldmann** [comment]
```
    wheezy32:   File "/vagrant/borg/borg/.tox/py36/lib/python3.6/site-packages/borg/fuse.py", line 401, in process_leaf
    wheezy32:     source = os.path.join(*item.source.split(os.sep)[stripped_components:])
    wheezy32: TypeError: join() missing 1 required positional argument: 'a'
    wheezy32: 
    wheezy32: Platform: Linux vagrant 3.2.0-4-686-pae #1 SMP Debian 3.2.93-1 i686 
    wheezy32: Linux: debian 7.11 
    wheezy32: Borg: 1.1.3  Python: CPython 3.6.0
```

**Comment by ThomasWaldmann** [comment]
Happens if os.path.join() gets no arguments because *empty_list.

**Comment by ThomasWaldmann** [comment]
fixed in 1.1-maint by:
https://github.com/borgbackup/borg/pull/3391/commits/dc8de361095eea509a28d416ab826850047432ce
